exonum/src/helpers/fabric/details.rs

```diff
@@ -87,11 +89,11 @@ impl Command for Run {
                 false
             ),
             Argument::new_named(
-                "ROCKSDB_PATH",
+                DATABASE_PATH,
                 true,
-                "Use rocksdb database with the given path.",
+                "Use database with the given path.",
                 "d",
-                "rocksdb",
+                "database",
```

Comment:
Just a nitpick: May be beter `long_name` will be: "database_path" ?

Reply:
Underscores in parameters look disgusting as for me. What about `db-path` ?